diff --git a/mksingle.py b/mksingle.py
index eb336ce..d651d43 100755
--- a/mksingle.py
+++ b/mksingle.py
@@ -253,8 +253,9 @@ def main(argv):
   except OSError:
     pass
 
-  zf = zipfile.ZipFile(zip_output_file_name, 'w', zipfile.ZIP_DEFLATED)
-  time_now = time.localtime()[:6]
+  buf = cStringIO.StringIO()
+  zf = zipfile.ZipFile(buf, "w", zipfile.ZIP_DEFLATED)
+  time_now = time.localtime(315554400)[:6]
   try:
     for file_name in (
         # 'pdfsizeopt/pdfsizeopt_pargparse.py',  # Not needed.
@@ -265,7 +266,7 @@ def main(argv):
       code_orig = open('lib/' + file_name, 'rb').read()
       # The zip(1) command also uses localtime. The ZIP file format doesn't
       # store the time zone.
-      file_mtime = time.localtime(os.stat('lib/' + file_name).st_mtime)[:6]
+      file_mtime = time.localtime(315554400)[:6]
       code_mini = MinifyFile(file_name, code_orig)
       # Compression effort doesn't matter, we run advzip below anyway.
       zf.writestr(new_zipinfo(file_name, file_mtime), code_mini)
@@ -281,11 +282,15 @@ def main(argv):
 
     file_name = 'pdfsizeopt/psproc.py'
     code_orig = open('lib/' + file_name, 'rb').read()
-    file_mtime = time.localtime(os.stat('lib/' + file_name).st_mtime)[:6]
+    file_mtime = time.localtime(315554400)[:6]
     code_mini = MinifyPostScriptProcsets(file_name, code_orig)
     zf.writestr(new_zipinfo(file_name, file_mtime), code_mini)
   finally:
     zf.close()
+  with open(zip_output_file_name, 'w') as fd:
+    import shutil
+    buf.seek(0)
+    shutil.copyfileobj(buf, fd)
 
   subprocess.check_call(('advzip', '-qz4', '--', zip_output_file_name))
 
