diff --git a/lib/pdfsizeopt/main.py b/lib/pdfsizeopt/main.py
index 946d6ab..48b2791 100644
--- a/lib/pdfsizeopt/main.py
+++ b/lib/pdfsizeopt/main.py
@@ -9437,8 +9437,8 @@ IMAGE_OPTIMIZER_CMD_MAP = {
     'optipng':  'optipng %(sourcefnq)s -o4 -fix -force %(optipng_gray_flags)s-out %(targetfnq)s',
     'optipng4': 'optipng %(sourcefnq)s -o4 -fix -force %(optipng_gray_flags)s-out %(targetfnq)s',
     'optipng7': 'optipng %(sourcefnq)s -o7 -fix -force %(optipng_gray_flags)s-out %(targetfnq)s',  # Slowest.
-    'ect': 'ECT -9 -strip %(targetfnq)s',
-    'ECT': 'ECT -9 -strip %(targetfnq)s',
+    'ect': 'ect -9 -strip %(targetfnq)s',
+    'ECT': 'ect -9 -strip %(targetfnq)s',
     'advpng':  'advpng -z3 -f %(targetfnq)s',
     'advpng3': 'advpng -z3 -f %(targetfnq)s',
     'advpng4': 'advpng -z4 -f %(targetfnq)s',  # Slowest, this uses Zopfli.
